### PR TITLE
Change get pipeline sensitive variables from Vault instead of GitLab settings

### PR DESCRIPTION
**Pull request by sergejparity**
Change get pipeline sensitive variables from Vault instead of GitLab settings

### CODE REVIEW

**Comment by rcny** [comment]
AFAIU this can't be merged yet. We still don't know if it works on `master`.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -223,6 +223,9 @@ build-nightly:
     GIT_STRATEGY:                  none
     DOCKERFILE:                    ci.Dockerfile
     IMAGE_NAME:                    docker.io/paritytech/$CI_JOB_NAME
+    VAULT_SERVER_URL: "https://vault.parity-mgmt-vault.parity.io"
+    VAULT_AUTH_PATH: "gitlab-parity-io-jwt"
+    VAULT_AUTH_ROLE: "cicd_gitlab_parity_${CI_PROJECT_NAME}"
```
Comment (tomusdrw) [inline .gitlab-ci.yml:228]: Please either fix alingment in the added lines or get rid of the alignment in others to keep it consistent.

**Review by tomusdrw — COMMENTED** [review]
Out of curiosity what's wrong with GitLab settings?

File: .gitlab-ci.yml (hunk 1 above)
Comment (sergejparity) [reply .gitlab-ci.yml]: Formatting fixed.

What goes to GitLab settings vs Vault:
For example secrets stored in GitLab are non-auditable and non-versionable.

**Review by TriplEight — APPROVED** [review]
Please make sure it works well on master after the merge.

**Pull request merged**